packages/tools/client-debugger/client-debugger-chrome-extension/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@fluid-tools/client-debugger-chrome-extension",
+  "version": "0.1.0",
+  "private": true,
+  "description": "A Chrome extension visual debugger for the Fluid Client.",
+  "homepage": "https://fluidframework.com",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/FluidFramework.git",
+    "directory": "packages/tools/client-debugger/client-debugger-chrome-extension"
+  },
+  "license": "MIT",
+  "author": "Microsoft and contributors",
+  "scripts": {
+    "build": "npm run build:genver && concurrently npm:build:compile npm:lint",
+    "build:compile": "npm run build:tsc && npm run webpack",
+    "build:full": "npm run build",
+    "build:full:compile": "npm run build:compile",
+    "build:genver": "gen-version",
+    "build:tsc": "npm run tsc",
+    "build:webpack": "npm run webpack",
+    "clean": "rimraf coverage dist nyc *.tsbuildinfo *.build.log",
+    "eslint": "eslint src",
+    "eslint:fix": "eslint src --fix",
+    "format": "npm run prettier:fix",
+    "format-and-build": "npm run format && npm run build",
+    "format-and-compile": "npm run format && npm run build:compile",
+    "lint": "npm run prettier && npm run eslint",
+    "lint:fix": "npm run prettier:fix && npm run eslint:fix",
+    "mocha": "mocha dist/**/*.test.js --unhandled-rejections=strict --exit",
+    "prettier": "prettier --check .",
+    "prettier:fix": "prettier --write .",
+    "start:tinylicious": "tinylicious",
+    "test": "npm run test:realsvc",
+    "test:coverage": "start-server-and-test start:tinylicious 7070 test:realsvc:coverage",
+    "test:realsvc": "start-server-and-test start:tinylicious 7070 test:realsvc:run",
+    "test:realsvc:coverage": "npm run test:realsvc:run:coverage",
+    "test:realsvc:run": "jest --detectOpenHandles",
+    "test:realsvc:run:coverage": "jest  --detectOpenHandles --coverage --ci --reporters=default --reporters=jest-junit",
+    "tsc": "tsc --pretty",
+    "webpack": "webpack --env production",
+    "webpack:dev": "webpack --env development"
+  },
+  "dependencies": {
+    "@fluid-tools/client-debugger": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
+    "@fluid-tools/client-debugger-view": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
```

Comment:
All `">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0"` need to be changed and updated. to `">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0"` In fact, I'm pretty sure since this is next, they may need to be updated to `">=2.0.0-internal.3.0.0 <2.0.0-internal.4.0.0"`

Reply:
This is blocking the main-next integrate to fail https://github.com/microsoft/FluidFramework/pull/13567

Reply:
@Josmithr @jikim-msft